SideBar.py

```diff
@@ -90,7 +90,73 @@ def copy(self, source, destination):
 			shutil.copy2(source, destination)
 
 		if self.view:
-			self.view.erase_status('ZZZ')
+			self.view.erase_status('SideBarTools:Copy')
 
 	def description(self):
 		return 'Duplicate File…'
+
+
+def temporary_status_message(view, message, key='SideBarTools', duration=5000):
```

Comment:
Not sure if there are docs about this, but I did the same thing [here](https://github.com/psykzz/st3-gitblame/commit/3665d7e88762309e8858a9bb34b641ce72588791#diff-1d1ddf0a0df5cb916ef6d515de593e37R110) and it works like a charm.